packages/tslint-config/README.md

```diff
@@ -0,0 +1,4 @@
+tslint-config
+-----------------------
+
+0xProject TypeScript Style Guide
```

Comment:
Maybe something like "Lint rules related to 0xProject for TSLint." This is the same description format used here: https://github.com/palantir/tslint-react.